config/shared/README.md

```diff
@@ -4,6 +4,8 @@
 
 The locales currently supported by the app are specified in [locales.json](./locales.json). Add a language there if you want to support a new translation.
 
+To expose locale in production, set `production:true`. Threshold for publishing is ~50% translated status. Don't un-publish translations from production because members might be using those locales.
```

Comment:
Just made the 50% up here and open any other suggestions. Typically this would be higher to have good experience, but consider you currently even can't translate like half the site, this seemed to strike good balance with showing at least _something_ translated.